.claude/skills/figma-v3-migration-plugin-dev/SKILL.md

```
@@ -0,0 +1,360 @@
+name: figma-v3-migration-plugin-dev
+description: Figma V3 Migration Plugin development specialist. Use when developing Figma V3 Migration Plugin. Focuses on Figma V3 Migration Plugin development.
+allowed-tools: Read, Write, Edit, MultiEdit, Bash, Glob, Grep
+---
+
+## 개요
+
+Figma V3 Migration 플러그인은 SEED Design System V2 컴포넌트를 V3로 마이그레이션하는 Figma 플러그인입니다.
+
+### 디렉토리 구조
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)